libcxx/test/std/containers/views/mdspan/mdspan.extents.cons/span.pass.cpp

```diff
@@ -19,7 +21,7 @@ void test_span_con()
     using TestFixture = TestExtents<T>;
     TestFixture t;
 
-    auto s = std::span( t.dyn_sizes );
+    auto s = std::span<const size_t, t.dyn_sizes.size()>( t.dyn_sizes );
```

Comment:
@youyu3 If you're using `std::span`, you'll need C++20.  Would you consider using `cuda::std::span` (if it exists) instead?

Reply:
This test is for testing libcudacxx in the libcxx mode.  `std::span` is implemented in this mode.